### ISSUE TITLE
Ensures buttons have discernible text (a[role="button"]:nth-child(2)).

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#146]
User experience:
If, the buttons does not have discernible text that clearly describes the destination, purpose, function, or action for screen reader users, then the user who depends upon Screen Reader for navigation might get affected hence failed to get the proper information of the button.

Note: User credentials should NOT be included in the bug.

Pre-requisites:

1. Open Narrator via pressing ctrl+window+enter key. 
2. Open Accessibility insights for web tool and select fast pass option.

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Card and select it.
8.Navigate to the "Card" dialog and select edit button.
9. Navigate to the various controls of the "Card" dialog box content alignment, content flow etc.
10. Open accessibility insights for web tool and select fast pass option.
11. Verify, whether the button does have discernible text defined or not.

Actual Result:

On the card dialog box, buttons available below content alignment control does not have discernible text defined, due to which Screen Reader announce button only.

Expected Result:

On the card dialog box, buttons available below content alignment control should have discernible text defined, so that the Screen Reader user does able to get proper information of the button.

Note:1. This issue also exist on the edge browser.
         2. This issue also exist for the button available below vertical controls.   
 
MAS Reference:

MAS 4.1.2 - Name, Role, Value

Issue: Ensures buttons have discernible text (button-name - https://dequeuniversity.com/rules/axe/3.3/button-name?application=msftAI)

Target application: API Management - https://accessibility.developer.azure-api.net/apis

Element path: .form-group:nth-child(1) > a[role="button"]:nth-child(2)

Snippet: <a href="#" class="toolbox-btn" role="button"> <i class="paperbits-icon paperbits-align-left"></i> </a>

How to fix: 
Fix any of the following:
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"
Element has no title attribute or the title attribute is empty

This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights. 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

![image](https://user-images.githubusercontent.com/2320302/80266042-c8a44000-864e-11ea-903d-f5a9ac4f2c0b.png)

**Comment by azaslonov** [comment]
Fixed.
